modin/core/io/text/text_file_dispatcher.py

```diff
@@ -641,6 +649,24 @@ def check_parameters_support(
         if read_kwargs["chunksize"] is not None:
             return False
 
+        skiprows_unsupported = False
```

Comment:
Double negatives are hard to read.

```suggestion
        skiprows_supported = True
```